keyboards/nifty_numpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 Isaac Rex (@Acliad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Layers 
+// NOTE: LAYER_RGB is defined in nifty_numpad.h
+enum LAYERS {
+    LAYER_BL,
+};
+
+// Setup keymap
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * SCN -> Screenshot
+     * WIN -> Task View
+     * EXP -> Open Explorer
+     * BCK -> Backspace
+     * ┌───┬───┐  ┌───┬───┬───┬───┐
+     * │F13│F19│  │BCK│WIN│EXP│SCN│
+     * └───┴───┘  └───┴───┴───┴───┘
+     * ┌───┬───┐  ┌───┬───┬───┬───┐
+     * │F14│F20│  │Num│ / │ * │ - │
+     * ├───┼───┤  ├───┼───┼───┼───┤
+     * │F15│F21│  │ 7 │ 8 │ 9 │   │
+     * ├───┼───┤  ├───┼───┼───┤ + │
+     * │F16│F22│  │ 4 │ 5 │ 6 │   │
+     * ├───┼───┤  ├───┼───┼───┼───┤
+     * │F17│F23│  │ 1 │ 2 │ 3 │   │
+     * ├───┼───│  ├───┴───┼───┤Ent│
+     * │F18│F24│  │ 0     │ . │   │
+     * └───┴───┘  └───────┴───┴───┘
+     */
```

Comment:
These commented layout representations typically always fall out of date with the actual keymap.
```suggestion
```